examples/python/AutoCalibration/auto_calibration_example.py

```
@@ -162,14 +121,7 @@ def botchCalibration(device : dai.Device):
                 print("Did not pass")
 
         depth = stereoOut.get()
-        showDepth(
-            depth.getCvFrame(),
-            windowName="Depth",
-            minDistance=500,
-            maxDistance=5000,
-            colormap=cv.COLORMAP_TURBO,
-            useLog=False
-        )
+        cv.imshow("Depth", dai.colorizeDepthFrame(depth, 500, 12000, cv.COLORMAP_TURBO, False).getCvFrame())
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)